pkg/keptn/errorlog.go

```diff
@@ -96,34 +102,20 @@ func (els *ErrorLogSender) SendErrorLogEvent(initialCloudEvent *cloudevents.Even
 	return fmt.Errorf("no registration found with name %s", els.integrationName)
 }
 
-func createErrorLogCloudEvent(
-	integrationID string, initialEvent *cloudevents.Event, err error,
-) (cloudevents.Event, error) {
-	errorData := ErrorData{
+func createErrorLog(
+	integrationID string, initialEvent *sdk.KeptnEvent, err error,
+) (models.LogEntry, error) {
```

Comment:
It seems like the function never returns an error

Reply:
It seems like I forgot to change the return types. I fixed it now.